server/notification-providers/pushover.js

```diff
@@ -23,6 +26,11 @@ class Pushover extends NotificationProvider {
             "html": 1,
         };
 
+        const baseURL = await setting("primaryBaseURL");
+        if (baseURL && monitorJSON) {
+            data["url"] = baseURL + getMonitorRelativeURL(monitorJSON.id);
```

Comment:
I think that link might be somewhat confusing.
What do you think about adding a `url_title`?

```suggestion
            data["url"] = baseURL + getMonitorRelativeURL(monitorJSON.id);
            data["url_title"] = "Link to Monitor"
```

Reply:
works for me, will add semi colon

```suggestion
            data["url"] = baseURL + getMonitorRelativeURL(monitorJSON.id);
            data["url_title"] = "Link to Monitor";
```